packages/core/template/portfolio/contents/experience.md

```diff
@@ -0,0 +1,25 @@
+# Experience
+
+<panel type="seamless" no-close minimized>
+  <div slot="header">
+    <md>**Software Engineering Intern @ [NUS](https://nus.edu.sg/)** | May 2024 - Aug 2024</md>
```

Comment:
Hmmm the links here don't work because the panel will always open instead

Reply:
Hi I've changed the panels for this section to expanded so that its easier to click open the links.

Reply:
Yep I've since changed the look of the experience section, inspired by the "Where I've worked" section on @ckcherry23 website. It now uses tabs and the company is linked in the header of the tab content. You can check it out here: https://portfolio-template-markbind.netlify.app/